app/models/content_item.rb

```diff
@@ -2,7 +2,7 @@ class ContentItem
   include Mongoid::Document
   include Mongoid::Timestamps::Updated
 
-  NON_RENDERABLE_FORMATS = %w(redirect gone)
+  NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
```

Comment:
Yup - though I've realised it's just in a (2 year old) [migration](https://github.com/alphagov/content-store/blob/master/db/migrate/20141126123135_add_locale_to_existing_content.rb)

So I imagine it is safe to kill.